plantio/models/plantio.py

```diff
@@ -15,3 +21,18 @@ class Plantio(models.Model):
     talhao = models.ForeignKey(Talhao, on_delete=models.PROTECT)
     dataPlantio = models.DateField()
     estado = models.CharField(max_length=10, choices=PLANTIO_CHOICES)
+    agrotoxico = models.ManyToManyField(Agrotoxico, through='AssociacaoPlantioAgrotoxico')
+
+
+class AssociacaoPlantioAgrotoxico(models.Model):
+    plantio = models.ForeignKey(Plantio, on_delete=models.PROTECT)
+    agrotoxico = models.ForeignKey(Agrotoxico, on_delete=models.PROTECT)
+    dataAplicacao = models.DateField()
+    # TODO: Implementação de foto (Azure) -> issue 152
+    # caminhoFotoAgrotoxico = models.CharField(max_length=255)
+    dosagemAplicacao = models.DecimalField(
+        max_digits=3, decimal_places=2, validators=[MinValueValidator(Decimal('0.01'))])
+    estadoAnalise = models.CharField(max_length=1, choices=ESTADOS_CHOICES, default='A')
+
+    class Meta:
+        unique_together = [('plantio', 'agrotoxico', 'dataAplicacao')]
```

Comment:
acho que não tem como ter esse unique porque não se sabe o agrotoxico em tempo criação. Ele só sera adicionado dps em analise pelo tecnico ou se o produtor souber qual ele aplicou

Reply:
Pelo que eu entendi, você fez esse comentário imaginando o agrotóxico como opcional, certo ? Se for faz sentido seu comentário, mas para a modelagem dessa tabela na minha opnião deve ser obrigatorio logo a unique funciona bem.